docs/customization_checklist.md

```diff
@@ -1 +1,40 @@
-Want to use this for your own event? Here's what you need to change:
+## Welcome to EUI Event Template customization guide 
+
+If you want to use this template for your own event, here is everything that you need to change along with a brief description of how to make the respective change. The guide has been divided by sections of the web app:
+
+### Nav Bar:
+
+- [ ] **Change logo and favicon:** Follow the path ```eui-event-template/src/data/consts.js```. In the ```const.js``` file, on _line 12_, replace the placeholder rainbow cluster link with a link of your event’s logo.
```

Comment:
What do you think about leaving a permalink here to reference that line? Reference this link: 
https://github.com/brittanyjoiner15/eui-event-template/blob/8cefe4c73b1dea7f5fa33c6bd15071cb1bbae463/src/data/consts.js#L12